db/migrate/20191101162252_remove_unused_columns.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveUnusedColumns < ActiveRecord::Migration[6.0]
+  def change
+    remove_column :auctions, :image
```

Comment:
At this point, I think you could likely just modify the original migration instead of adding a new, since these migrations have not been applied anywhere other than your local database.

Once I migration has been merged to master, generally don't update it, but for a migration in a pull request, feel free to update it.